vunit/ui/library.py

```diff
@@ -290,11 +291,25 @@ def package(self, name):
 
         return PackageFacade(self._parent, self._library_name, name, design_unit)
 
-    def entity(self, name):
+    def get_entity(self, name):
         """
         Get an entity within the library
+        """
+        library = self._project.get_library(self._library_name)
```

Comment:
Maybe lines 298-302 can be moved to a _helper function to avoid duplication (284-288)?